zmsadmin/src/Zmsadmin/OverallCalendar.php

```
@@ -19,26 +19,13 @@ public function readResponse(
-                '/scope/' . $scopeId . '/department/',
-                ['resolveReferences' => 2]
-            );
-
-            $data = $departmentApiResult ? $departmentApiResult->getEntity() : null;
-            if ($data) {
-                $department = new DepartmentEntity($data);
-                $departmentScopes = $department->getScopeList();
-            }
+        $workstation = $result->getEntity();
+        if (!$workstation->getUseraccount()->hasRights(['scope'])) {
+            throw new \BO\Zmsentities\Exception\UserAccountMissingRights();
```

PHPMD (2.15.0)

Missing class import via use statement (line '28', column '23'). (Clean Code Rules)

(MissingImport)